### DISCUSSION TITLE
Feature request: official NodeJS implementation 🙏

### DISCUSSION THREAD

**Discussion by wholenews** [explosion/spaCy#8071] [New Features & Project Ideas]
The javascript community humbly begs your support!

We shower praise on https://github.com/huggingface/node-question-answering and hope for spaCy too.

- Operating System:
- Python Version Used:
- spaCy Version Used:
- Environment Information:

**Comment by svlandeg** [comment]
Hi!

Apologies for the late follow-up. Also I'm afraid we don't really have this on our radar. But I'll transfer this issue to the ["Project ideas"](https://github.com/explosion/spaCy/discussions/categories/new-features-project-ideas) section of our discussion forum - where hopefully someone may pick it up!
